tests/arc/trepr.nim

```diff
@@ -71,3 +71,19 @@ proc p2 =
 
 discard repr p2
 
+
+#####################################################################
```

Comment:
move to tests/arc/tarcmisc.nim
(not related to repr IIUC)

Reply:
I have`trepr` because is busy `tarcmisc` with my other PR and I don't want to introduce merging conflicts. Anything else except `tarcmisc` ?